webpack/webpack.dev.js

```diff
@@ -0,0 +1,18 @@
+const webpack = require('webpack');
+const ReactRefreshWebpackPlugin = require('@pmmmwh/react-refresh-webpack-plugin');
+
+module.exports = {
+    mode: 'development',
+    devServer: {
+        hot: true,
+        open: true
+    },
+    devtool: 'cheap-module-source-map',
+    plugins: [
+        new webpack.DefinePlugin({
+            'process.env.name': JSON.stringify('Dev')
```

Comment:
what the point write string with `JSON.stringify`? Sorry, but I didn't work out it.